README.md

```diff
@@ -53,8 +56,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+The big-O notation to validate a path configuration is O(n/2), but since big-0 notation ignores constants, it is O(n). This is the case because it will only need to compare one half of the word with the other half of the word, so it would work in from each end comparing the two letters to make sure they are the same. 
```

Comment:
# 3

In this particular case, what you say is true (although the O(n/2) is spurious, as it would ned the whole string). But a general purpose parse based on the grammar would use a stack, and that stack would also be O(n).